cpp/src/dataframe/types.hpp

```diff
@@ -11,6 +11,7 @@
 
 /* --------------------------------------------------------------------------*/
 /** 
+ * @file types.hpp
  * @brief  Wrapper structs for for the non-fundamental gdf_dtype types.
```

Comment:
You don't need a `@brief` after a `@file`, IIANM.

Reply:
From the Doxygen Manual they use a `brief` after a `file`: ```/*! \file structcmd.h
    \brief A Documented file.
    
    Details.
*/```

Reply:
Right you are.